libraries/dm/log_appender.cpp

```diff
@@ -0,0 +1,61 @@
+#include <dfuse/dm/log_appender.hpp>
+#include <fc/log/log_message.hpp>
+#include <fc/log/logger_config.hpp>
+#include <fc/string.hpp>
+#include <fc/variant.hpp>
+#include <fc/reflect/variant.hpp>
+#ifndef WIN32
+#include <unistd.h>
+#endif
+#include <boost/thread/mutex.hpp>
+#include <fc/exception/exception.hpp>
+#include <iomanip>
+#include <mutex>
+#include <sstream>
+
+namespace dfuse { namespace dm {
+   class log_appender::impl {
```

Comment:
I think this could be moved into `fc` as a type of registered appender like `console` and `gelf`. Then you can configure it via a `logging.json`. Call it say `dmlog` type since the `DMLOG` is hard coded.

We hope to move to a more full featured logging framework in the future so this hopefully would only be temporary.

Reply:
It was in `fc` before, but it was harder for us to maintained, since we had two private forked submodules, which is not easy to deal with.

So we moved it back directly in EOSIO. I don't have strong disagreement here, I'm willing to move it back to `fc` again.

`dmlog_appender` would be correct with you?

Reply:
yes.